<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WLW7-KavithaLN" name="Test B1" tests="3" failures="1" timestamp="17 Jul 2014 13:06:32 GMT" time="0.004" errors="0">
  <testcase name="testB1_1" time="0.0" classname="testSuiteB.TestB1">
    <skipped/>
  </testcase> <!-- testB1_1 -->
  <testcase name="testB1_2" time="0.0" classname="testSuiteB.TestB1"/>
  <testcase name="testB1_3" time="0.0" classname="testSuiteB.TestB1">
    <failure type="java.lang.AssertionError" message="failed test case">
      <![CDATA[java.lang.AssertionError: failed test case
	at testSuiteB.TestB1.testB1_3(TestB1.java:22)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testB1_3 -->
</testsuite> <!-- Test B1 -->
